.ci/containers/gcb-terraform-vcr-tester/test_terraform_vcr.sh

```diff
@@ -231,7 +232,8 @@ if [[ -n $FAILED_TESTS_PATTERN ]]; then
   else
     if [[ $test_exit_code -ne 0 ]]; then
       # check for any uncaught errors in RECORDING mode
-      comment+="Errors occurred during RECORDING mode. Please fix them to complete your PR${NEWLINE}"
+      comment+="Errors occurred during RECORDING mode. Please [verify if they are related](https://github.com/GoogleCloudPlatform/magic-modules#verifying-failed-recording-tests) to your changes and fix them to complete your PR${NEWLINE}"
+      comment+="It is recommended to [build the provider](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers) to verify locally.${NEWLINE}"
```

Comment:
Do we also want to add the similar comment after line 231? It is the comment where lists the failed tests for every PR.

Reply:
Oh, I think i mixed up the two cases. Yeah I'll move the comment around

Reply:
I think we may also want to keep this comment here as well? Based on current setup: VCR may failed in recording in three cases:
- provider crash
- failed tests
- uncaught errors 

Maybe we want this comment generated in all three cases?

Reply:
I wanted to ask them to verify the failed tests for the failed tests case, but then tell them to build the provider for provider crashed/uncaught errors. Instead of saying both verify failed tests AND build the provider for all 3 cases. What do you think?

Reply:
Yeah, sounds good! It' pretty clear! Sorry I just missed that you kept the build provider part in the uncaught error comment. Sorry for the confusion!